variables.tf

```
@@ -174,6 +174,26 @@ variable "oidc" {
   default = null
 }
 
+variable "automatic_filesystem_trim" {
+  description = "Settings to enable and configure automatic filesystem trim of volumes managed by Longhorn."
+  type = object({
+    enabled   = bool
+    cron      = string
+    job_group = string
+  })
+  default = {
+    enabled   = false
+    cron      = "0 6 * * *"
+    job_group = ""
+  }
+}
+
+variable "recurring_job_selectors" {
+  description = "Define a group list to add to recurring job selector for the default storageclass (the custom backup one if `set_default_storage_class` is set or else the Longhorn default one)"
+  type        = string
+  default     = ""
```

It seems to me that this variable is always static and the expected format is JSON. Do you think we can use a properly structured Terraform variable and then encode it in JSON to pass to the values?

Something like this:

```suggestion
  type = map(object({
    name    = string
    isGroup = bool
  }))
  default = null
```

Then in the line 75 of the `locals.tf` you would do:

```
recurringJobSelector = var.automatic_filesystem_trim.enabled && var.enable_pv_backups && var.set_default_storage_class ? jsonencode(var.recurring_job_selectors) : null
```

I've not tested it, but it should work :stuck_out_tongue_winking_eye: